hyundai_kia_connect_api/KiaUvoApiEU.py

```diff
@@ -598,9 +613,13 @@ def _update_vehicle_properties_ccs2(self, vehicle: Vehicle, state: dict) -> None
         vehicle.air_control_is_on = get_child_value(
             state, "Cabin.HVAC.Row1.Driver.Blower.SpeedLevel"
         )
-        # TODO: vehicle.smart_key_battery_warning_is_on = get_child_value(
-        # TODO:     state, "status.smartKeyBatteryWarning"
-        # TODO: )
+        smart_key_battery_warning_is_on = get_child_value(
```

Comment:
This if-else can be better written as:
````
        smart_key_battery_warning_is_on = get_child_value(
            state, "Electronics.FOB.LowBattery"
        ) != 0
````

Reply:
As python newbie I realized that I even can use the bool function :-)